sklearn/utils/validation.py

```diff
@@ -866,21 +866,18 @@ def check_symmetric(array, tol=1E-10, raise_warning=True,
     return array
 
 
-def check_is_fitted(estimator, attributes, msg=None, all_or_any=all):
+def check_is_fitted(estimator, *, msg=None):
```

Comment:
aren't you changing the signature here? did you mean 

`check_is_fitted(estimator, *args, msg=None):` to preserve backward compatibility?

Reply:
I'm changing the signature. We could have ``args`` here if we consider this public, which maybe we should?

Reply:
ok actually using ``args`` doesn't work unless I also add ``*kwargs``. So if we want backward-compatibility we need to just do a usual deprecation cycle.

Reply:
Yes I think we consider the validation utils public. But I'm happy to see them go private.